tests/channels-search.bats

```diff
@@ -41,6 +40,19 @@ teardown() {
 
 setup_file() {
   export FLOX_FEATURES_CHANNELS=rust;
+
+  # Necessary to make search results consistent for local dev and CI
+  # This variable sets `Flox.system`, which we pass along to `pkgdb`
+  # in the "systems" search parameter.
+  export NIX_TARGET_SYSTEM=x86_64-linux;
+
+  # Separator character for ambiguous package sources
+  export SEP=":";
+
+  # The current system
+  _os=$(uname -s)
+  _cpu=$(uname -m)
+  export THIS_SYSTEM="$_cpu-${_os,,}"
```

Comment:
You can drop this and use `$NIX_SYSTEM` ( `x86_64-linux`, `aarch64-darwin`, etc ).
It's set by setup_suite.bash:198.